docs/docs/troubleshooting.md

```diff
@@ -67,6 +67,12 @@ Since then, a much simpler procedure of performing a bluetooth reset for split k
 After completing these steps, pair the halves of the split keyboard together by resetting them at the same time. Most commonly, this is done by grounding the reset pins
 for each of your keyboard's microcontrollers or pressing the reset buttons at the same time.
 
+### macOS Pairing Issues
+
+Sometimes, macOS will _say_ your board is connected, but no keypresses will be sent. Double check that everything else is working by plugging the main side in via USB; if both sides register keypresses, i.e., they are connected to each other, you may be affected by the trash stack.
```

Comment:
Since the #946 enhancement is now in main, is this section still relevant @muppetjones?